mne/decoding/time_delaying_ridge.py

```diff
@@ -67,9 +74,8 @@ def _compute_corrs(X, y, smin, smax):
             for idx in range(1, smax):
                 ii = idx - smin
                 end_sl = slice(X.shape[0] - idx, -smax - min(ii, 0), -1)
-                c = (this_X[-idx, other_sl][np.newaxis] *
-                     this_X[end_sl, ch0][:, np.newaxis])
-                r = this_X[-idx, ch0] * this_X[end_sl, other_sl]
+                c = this_X[-idx, that_sl] * this_X[end_sl, this_sl]
+                r = this_X[-idx, this_sl] * this_X[end_sl, that_sl]
```

Comment:
Idk, but maybe even marginally faster/cleaner doing:

```
this_X_this = this_X[:, this_sl]
this_X_that = this_X[:, that_sl
for idx in range(1, smax):
...
    c = this_X[-idx, that_sl] * this_X_this[end_sl]
...
```

See what I mean?

Reply:
I'm not sure it's more readable. It breaks the symmetry.

In terms of speed, I assume it's negligible because it's just creating a view but I have not benchmarked it

Reply:
Yeah it's nothing important.

As you see, I have no meaningful comments :D